upack/Install.cs

```diff
@@ -48,33 +48,85 @@ public sealed class Install : Command
         [DefaultValue(false)]
         public bool Prerelease { get; set; } = false;
 
+        [DisplayName("comment")]
+        [Description("The reason for installing the package, for the local registry.")]
+        [ExtraArgument]
+        public string Comment { get; set; }
+
+        [DisplayName("userregistry")]
+        [Description("Cache the package in the user registry instead of the machine registry.")]
+        [ExtraArgument]
+        [DefaultValue(false)]
+        public bool UserRegistry { get; set; } = false;
+
+        [DisplayName("unregistered")]
+        [Description("Do not cache the package in a local registry.")]
+        [ExtraArgument]
+        [DefaultValue(false)]
+        public bool Unregistered { get; set; } = false;
+
```

Comment:
Since we do have file caching implementing, we need to add an option to "opt-in" to it. it's supposed to be disabled by default...

[ExtraArgument]
[DefaultValue(false)]
public bool CachePackages { get; set; } = false;